ceph/cephfs/auth/openshift-clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: cephfs-provisioner-runner
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "create", "delete"]
```

Comment:
create a role and rolebinding object without a namespace, you pass the namespace in when creating them with `kubectl create -f ... --namespace secret-namespace`